met-web/src/services/contactService/index.tsx

```diff
@@ -0,0 +1,52 @@
+import http from 'apiManager/httpRequestHandler';
+import Endpoints from 'apiManager/endpoints';
+import { replaceUrl } from 'helper';
+import { Contact } from 'models/contact';
+
+export const getContact = async (contactId: number): Promise<Contact> => {
+    const url = replaceUrl(Endpoints.Contacts.GET, 'contact_id', String(contactId));
+    if (!contactId || isNaN(Number(contactId))) {
+        return Promise.reject('Invalid Contact Id ' + contactId);
+    }
+    try {
+        const response = await http.GetRequest<Contact>(url);
+        if (response.data.result) {
+            return Promise.resolve(response.data.result);
+        }
+        return Promise.reject(response.data.message ?? 'Failed to fetch contact');
+    } catch (err) {
+        return Promise.reject(err);
+    }
+};
+
+export const getContacts = async (): Promise<Contact[]> => {
+    try {
+        const response = await http.GetRequest<Contact[]>(Endpoints.Contacts.GET_LIST);
+        if (response.data.result) {
```

Comment:
what case will be response.data.result be non-truthy? just curious

Reply:
Copying over code from other services here, I'm not sure either. @tiagograf-AOT any particular thoughts on this?

Reply:
added https://app.zenhub.com/workspaces/met-ops-6254279dcfa60b0010ea5a0d/issues/bcgov/met-public/760